backend/Main.js

```diff
@@ -0,0 +1,11 @@
+const express = require('express');
+const bodyParser = require('body-parser');
+
+const app = express();
+const userRouter = require('./user/User');
```

Comment:
That's a confusing folder structure right there, maybe routers/user? You can debate this.

Reply:
but its not router,its like Main but for user 
maybe i can rename to "main"  or "user-main" and it will be ```/user/main | user-main```